buddyboss/buddyboss-platform
# Excluded group type post type from including in query reset in instuc…

…tor role plugin

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1431 .

### How to test the changes in this Pull Request:

1. Install and Activate Instructor plugin
2. Create a Group Type
3. Add Group Role Labels for Organizer, Moderator, and Members.
4. Assign the Group Type you just created to any Group.
5. On the same group, set an Organizer with an Instructor Role.
6. Notice the error

### Proof Screenshots or Video

https://www.loom.com/share/db88de175d2c402cb3ed3fe9b49b91c6

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
